clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -205,7 +208,11 @@ private TachyonFS(TachyonConf tachyonConf) {
    */
   synchronized void accessLocalBlock(long blockId) throws IOException {
     if (mWorkerClient.isLocal()) {
-      mWorkerClient.accessBlock(blockId);
+      try {
+        mWorkerClient.accessBlock(blockId);
+      } catch (BlockDoesNotExistException e) {
```

Comment:
Should this just be ``TException e`?

That way if the implementation of accessBlock changes potentially throwing a different set of Thrift exceptions, we do not have to change this.

This applies to all locations where we convert a Thrift exception to an IO exception.